app/Schema/account_transaction_details_schema.py

```diff
@@ -48,16 +47,15 @@ class FundTransferSchema(ma.Schema):
 
     """Adding the Schema validations to the FundTransfer """
 
-    from_account = fields.Str(required=True, validate=Regexp("^[0-9]{10}$|^[0-9]{12}$"))
-    to_account = fields.Str(validate=Regexp("^[0-9]{10}$|^[0-9]{12}$"))
-    transaction_amount = fields.Int(required=True)
+    from_account = fields.Str(required=True, validate=(Regexp("^[0-9]{10}$|^[0-9]{12}$"), Length(equal=10)))
+    to_account = fields.Str(validate=(Regexp("^[0-9]{10}$|^[0-9]{12}$"), Length(equal=10)))
+    transaction_amount = fields.Int(strict=True, required=True, validate=Range(min=1000, max=50000))
```

Comment:
Why maximum if 50000?

Reply:
It's being taken 50k as maximum debit because some bank's debit card limit  is 10k, 20k, 50k per day maximum transaction limit, using that logic I have taken max transaction as 50k limit.